go.mod

```diff
@@ -5,97 +5,99 @@ go 1.17
 require (
 	github.com/aws/aws-sdk-go v1.38.41
 	github.com/backube/volsync v0.3.0
-	github.com/csi-addons/kubernetes-csi-addons v0.3.1-0.20220510101231-848e8b27d395
-	github.com/csi-addons/volume-replication-operator v0.3.1-0.20220623123415-de6a161ca0ed
+	github.com/csi-addons/kubernetes-csi-addons v0.5.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-logr/logr v1.2.3
 	github.com/google/uuid v1.3.0
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/gomega v1.20.1
 	github.com/open-cluster-management/api v0.0.0-20210527013639-a6845f2ebcb1
-	github.com/operator-framework/api v0.14.1-0.20220413143725-33310d6154f3
+	github.com/operator-framework/api v0.15.1-0.20220624132056-decf74800a17
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.1
+	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/stolostron/config-policy-controller v0.0.0-20220503151013-2a6d24b0ed07
 	github.com/stolostron/governance-policy-propagator v0.0.0-20220511204303-f1c56f6da1bf
 	github.com/stolostron/multicloud-operators-foundation v0.0.0-20220315092956-6dc184852d56
 	github.com/stolostron/multicloud-operators-placementrule v1.2.4-1-20220311-8eedb3f.0.20220411162042-3de0a2f908f1
-	go.uber.org/zap v1.21.0
-	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
-	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
-	k8s.io/api v0.23.6
-	k8s.io/apimachinery v0.23.6
+	go.uber.org/zap v1.23.0
+	golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90
+	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
```

Comment:
there are a few updates to use Kubernetes 0.25, not sure it can cause any problem, let @ShyamsundarR confirm on it.

Reply:
This is replaced below, 

This automatically done when csi-addons was updated.